api/swagger/swagger.yaml

```diff
@@ -180,7 +225,31 @@ definitions:
         type: string
       updatedAt:
         type: string
-        
+  EventListResponse:
+    required:
+      - events
+    properties:
+      events:
+        type: array
+        items:
+          type: object
+          properties:
+            title:
+              type: string
+            purpose:
+              type: string
+            yelpID:
+              type: string
+            capacity:
+              type: number
+            attendeesCount:
+              type: number
+            startTime:
+              type: string
+            authorName:
```

Comment:
there's no other name in the response, so just `author` or one word `username` would be more concise